staging/src/k8s.io/apiserver/plugin/pkg/audit/buffered/buffered.go

```diff
@@ -228,16 +242,18 @@ func (b *bufferedBackend) processEvents(events []*auditinternal.Event) {
 		return
 	}
 
-	// TODO(audit): Should control the number of active goroutines
-	// if one goroutine takes 5 seconds to finish, the number of goroutines can be 5 * defaultBatchThrottleQPS
 	if b.throttle != nil {
 		b.throttle.Accept()
 	}
 
-	b.wg.Add(1)
+	// control the maximum number of go routines
```

Comment:
Please change the formatting to conform with the rest of the file, i.e. the comment should start with a capital letter and end with a comma.

Reply:
ok